packages/lit-dev-content/samples/examples/react-basics/app.tsx

```diff
@@ -1,14 +1,14 @@
-import {React, ReactDOM} from './react.js';
+import React from 'https://esm.sh/react@18';
+import {createRoot} from 'https://esm.sh/react-dom@18/client';
 import {createComponent} from '@lit-labs/react';
 import {DemoGreeting as DemoGreetingWC} from './demo-greeting.js';
 
 const DemoGreeting = createComponent({
```

Comment:
Could you add a comment here like `// This creates a React component from a Lit component`?